extensions/3DTILES_implicit_tiling/README.md

```diff
@@ -0,0 +1,946 @@
+- [Subtree JSON Schema Reference](#subtree-json-schema-reference)
+- [Appendix A: Availability Indexing](#appendix-a-availability-indexing)
+
+## Overview
+
+**Implicit tiling** is a new representation of a Cesium 3D Tileset that allows for fast random access of tiles and enables new traversal algorithms.
+
+Implicit tilesets are uniformly subdivided into a quadtree or octree. This regular pattern allows the tileset to be expressed in a more compact representation which keeps the tileset JSON small.
+
+The `3DTILES_implicit_tiling` extension can be added to any tile in the tileset. This extension defines how the tile is subdivided and where to locate content resources for the implicit tileset.
+
+In order to support sparse datasets, **availability** data determines which tiles exist. To support massive datasets, availability is partitioned into fixed-size **subtrees**. Subtrees are stored in a compact binary file format.
+
+Implicit tiling allows directly accessing tiles by their **tile coordinates**: `(level, x, y)` for quadtrees or `(level, x, y, z)` for octrees. This avoids traversing the full path to the tile.
+
+<img src="figures/implicit-vs-explicit.jpg" width="600"/>
+
+For a complete list of terminology used, see the [Glossary](#glossary).
+
+## Use Cases
+
+_This section is non-normative_
+
+A key use for implicit tiling is enabling and/or accelerating tree traversal algorithms. Accessing a tile by coordinates is faster than traversing the entire tree. Likewise, raycasting algorithms and GIS algorithms can benefit from the abbreviated tree traversals. Tiles can be loaded immediately instead of going from top to bottom of a tree.
+
+Accessing tiles by coordinates also helps accelerate spatial queries. For example, the highest resolution tile that contains a given point can be quickly located by computing the coordinates of the tile directly. 
+
+Implicit tiling also allows for better interoperability with existing GIS data formats with implicitly defined tiling schemes. Some examples are:
+
+* [CDB](https://docs.opengeospatial.org/is/15-113r5/15-113r5.html)
+* [S2](http://s2geometry.io/)
+* [WMTS](https://www.ogc.org/standards/wmts)
+* [TMS](https://wiki.osgeo.org/wiki/Tile_Map_Service_Specification)
+
+## Tile Extension
+
+The `3DTILES_implicit_tiling` extension may be defined on any tile in the tileset JSON. Such a tile is called an **implicit root tile**, to distinguish it from the root tile of the tileset JSON. The implicit root tile must omit the `children` property.
+
+```json
+{
+  "asset": {
+    "version": "1.0"
+  },
+  "geometricError": 10000,
+  "extensionsUsed": [
+    "3DTILES_implicit_tiling",
+  ],
+  "extensionsRequired": [
+    "3DTILES_implicit_tiling",
+  ],
+  "root": {
+    "boundingVolume": {
+      "region": [-1.318, 0.697, -1.319, 0.698, 0, 20]
+    },
+    "refine": "REPLACE",
+    "geometricError": 5000,
+    "content": {
+      "uri": "content/{level}/{x}/{y}.b3dm"
+    },
```

Comment:
(I'll integrate the comments (also for the content-gltf one) soon, but a quick question on that one:) 

I'll have to re-scan the section and think more deeply about whether one could apply _any_ reasonable bounding volume here. Something like "a bounding volume of the highest level of the content" _might_ make sense for culling, to not send out the request.

So the question is: **SHOULD** it be omitted (under certain conditions?), or **MUST** it be omitted?

Reply:
Since the content uri refers to the collective set of contents I don't know if we want the content bounding volume to apply to only the root tile's content, though that is a use case I was thinking about as well.

I'm leaning towards **MUST** be omitted.

Similar to my comment above we can say that content bounding volumes may be defined with tile metadata and point to the appropriate semantics in the Cesium Metadata Semantic Reference.